Assets/Scripts/GivePlayerDamage.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GivePlayerDamage : MonoBehaviour
+{
+    public int damageAmount = 10;
+    public float cooldownTime = 2.0f;
+    private float nextDamageTime = 0.0f;
+
+    void OnTriggerEnter2D(Collider2D other)
```

Comment:
if there's a cooldown shouldn't this be an `OnTriggerStay2D` so you can't just wait in the spikes?

Reply:
that's how I wanted it if I'm understanding you correctly so you can have a second to sit in the spikes and not immediately get damaged again

Reply:
unless that means that it will never trigger again I see what you're saying

Reply:
then yeah could change it over to `OnTriggerStay2D`